lib/client.js

```diff
@@ -1,10 +1,13 @@
 /* global URL */
 import version from 'version'
 import { when, isObject, isString } from './utils'
+import IdleState from './utils/idleState'
 import Tracker from './tracker'
 import NativePromise from 'native-promise-only'
 
 const Promise = window.Promise || NativePromise
+
+export const IFRAME_SESSION_TIMEOUT = 5 * 60 * 1000 // Set timeout for iframe sessions to 5 minutes.
```

Comment:
That's correct @sarmadsangi! The host would manage the session timeout as the value can be configured by the host.

Reply:
(Nice to have) We could potentially pass this configuration value through the App Runtime options which would then send a `postMessage` to the apps which are associated with that App Runtime.

This would override the default value above.

Reply:
Addressed by https://github.com/zendesk/zendesk_app_framework_sdk/pull/145/commits/ff014214d0386310f8900cb2c2c515062f339a7a (hopefully).